jboss_wildfly/README.md

```diff
@@ -44,14 +44,14 @@ The JBoss/WildFly integration does not include any events.
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][8].
+Need help? Contact [Datadog support][5].
 
 
-[1]: https://developers.redhat.com/products/eap/overview/
-[2]: http://wildfly.org/
+[1]: https://developers.redhat.com/products/eap/overview
+[2]: http://wildfly.org
 [3]: https://github.com/DataDog/integrations-core/blob/master/jboss_wildfly/datadog_checks/jboss_wildfly/data/conf.yaml.example
```

Comment:
Can you fix this link usage? It's used wrongly above.

Reply:
done :)